src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's priority number in the client list.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPriority(String)}
+ */
+public class Priority {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Priority numbers should only contain characters L,M,H,U OR ulmh";
+    public static final String VALIDATION_REGEX = "[ulmhULMH]|^undefined$|^high$|^medium$|^low$";
```

Comment:
The ^$ might not be necessary, but I might be wrong
"[ulmhULMH]|undefined|high|medium|low" might work

Reply:
Hmm but it works now why change it ><